chia/wallet/wallet_nft_store.py

```diff
@@ -183,6 +196,7 @@ async def get_nft_list(
         if wallet_id is not None or did_id is not None:
             sql += " and"
         sql += " removed_height is NULL"
+        sql += " LIMIT %s OFFSET %s " % (count, start_index)
```

Comment:
it makes me nervous when we use string formatting and concatenation to generate SQL queries. Over time I think we should phase that out in favor of using parameterized sql statements.

Also, you use `%s` here, but `count` and `start_index` are integers, aren't they?
Given that python can't guarantee they be integers, I think this is actually quite risky, inviting sql injection attacks

Reply:
right, I do convert user input from RPC call into int so injection attack from that is not possible, but can convert into params in any case.